### PR TITLE
Add Member Group Picker

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Belongs to https://github.com/umbraco/UmbracoDocs/issues/1528

### CODE REVIEW

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Member-Group-Picker/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+versionFrom: 8.0.0
+---
+
+# Member Group Picker
+
+`Alias: Umbraco.MemberGroupPicker`
+
+`Returns: string`
+
+The member group picker opens a panel to pick one or more member groups from the member section. The value saved is of type string (comma separated ids).
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Member-Group-Picker/index.md:11]:
```suggestion
The Member Group Picker opens a panel to pick one or more member groups from the Member section. The value saved is of type string (comma separated IDs).
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Looks great @erikjanwestendorp !
Thanks adding this article 😄 

I've added a couple of suggestions - let me know if something doesn't add up!

**Comment by erikjanwestendorp** [comment]
@sofietoft Thanks for your review, I just committed your suggestions.

**Review by sofietoft — APPROVED** [review]
That was fast 😄 Thanks for applying the suggestions.

I'll get this merged right away!
Thanks again for pitching in 😄

**Pull request merged**